lib/bridge.js

```diff
@@ -201,18 +257,19 @@ Bridge.prototype.configAuth = function configAuth (enable) {
 };
 
 /**
- * This function returns all callbacks.
+ * This function returns all callback as instances or if urlsOnly is set only callbacks URLs.
  *
- * @returns {Promise<Callback[]>}
+ * @param {Boolean}   urlsOnly
+ * @returns {Promise}
  */
-Bridge.prototype.getCallbacks = function getCallbacks () {
+Bridge.prototype.getCallbacks = function getCallbacks (urlsOnly) {
```

Comment:
If true, the returned promise resolves to an array of objects with id and url, this is not what I would except if I set "urlsOnly" to true. We should either really return the url, or clearify the parameter name

Reply:
Renamed to `callbackUrlList`